docs/source/index.rst

```diff
@@ -27,10 +27,16 @@ This is the `Chainer <https://chainer.org>`_ documentation.
    comparison
    license
 
-
 Indices and tables
 ==================
 
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Other Support
```

Comment:
How about making a section "Community" and introducing other resources, including awesome-chainer?